docs/getting-started/learn-the-api.md

```diff
@@ -131,33 +131,118 @@ const isAuthenticated = useIsAuthenticated();
 
 The session is also available from the [Runtime](/reference/runtime/runtime-class.md) instance.
 
-## Services
+<!-- ## Plugins
```

Comment:
Is this expected to be commented or a mistake?

Reply:
Oh, I forgot about that.

When I commented that, my thinking was that it is kind of unnecessary to learn about that at this stage of the user journey. There's also similar information in the reference section. But the reference section doesn't has much visibility and I doubt most user will consult it.

Maybe I should move the commented documentation to a guide? In your opinion, does creating this guide would be a good investment of my time?

Reply:
I feel like the commented section is short enough that it can stay here without being a full guide. 

I think we can wait for a guide until we have more custom plugins done and clearer use cases.

Reply:
Not sure about staying there. As I mentionned, I dont wan't the user to focus on this at this stage of it's journey. The "Learn the API" goal is too showcased the most useful concepts of Squide and learning about plugin isn't one of them at this stage. Right now, the only thing that a user must know about plugisn is that he can register the MSW plugin. It will already be covered in the upcoming MSW guide.

I'll keep thinking about this, thanks for your feedback 🙏🏻

Reply:
Added a very brieg section about Plugins in the "Learn the API" page.